test/e2e/volume_rm_test.go

```diff
@@ -47,7 +47,7 @@ var _ = Describe("Podman volume rm", func() {
 		Expect(len(session.OutputToStringArray())).To(Equal(0))
 	})
 
-	It("podman rm with --force flag", func() {
+	It("podman volume rm with --force flag", func() {
```

Comment:
Is there a test with a volume that is removed that's associated with multiple containers?